src/components/home/index.js

```diff
@@ -39,6 +39,7 @@ class Home extends React.Component {
 
   componentWillUnmount() {
     this.visibilityTracker.removeEventListener(this.handleVisibilityChange);
+    this.props.clearInterval(this.refreshInterval);
```

Comment:
@ograu This is needed to stop the client from pulling data in the background, even when the user is no longer on the page.

Reply:
thanks @marians , I know what a `clearInterval()` do, I saw that we didn't cleared intervals here and actually noticed weeks ago that in Happa we have a lot of `setIntervals`/`setTimeouts` not cleared and, yeah, this is clearly something that has to be done.

I decided (maybe it was a wrong decision) to comment this refreshing because I was kind of thinking in redoing all of.

Reply:
OK. Just wanted to point it out since it must have gotten lost in some of the recent refactoring.